<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Set_IpWid</class>
 <widget class="QWidget" name="Set_IpWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string>IP-PDU后台参数设置区</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="rightMargin">
       <number>100</number>
      </property>
      <item row="1" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>安全加密</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QComboBox" name="securityBox">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>标准版本</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>安全加密</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QComboBox" name="ipModeBox">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>网络一带四</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Modbus RTU 级联</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="InFirst">
        <property name="text">
         <string>IEC309</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QComboBox" name="languageBox">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>英文</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>中文</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="ipTypeBox">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>IP-V1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>IP-V3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>IP-V3-C3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>IP-V3-EATON</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>IP-V6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>IP-V6-HUAWEI</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLineEdit" name="InSecond">
        <property name="text">
         <string>18A</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>设备版本</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>模式选择</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="lineBox">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>直流</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>单相</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>两路</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>三相</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>语言选择</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="logBox">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>无日志</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>带日志</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>日志功能</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>IN：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>相数选择</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>标准/中性</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>LCD方向</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="sBox">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>标准</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>中性</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>定制</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QComboBox" name="lcdBox">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>垂直</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>水平</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>垂直（翻转）</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>水平（翻转）</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>OUT:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="OutFirst">
        <property name="text">
         <string>20xC13</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QLineEdit" name="OutSecond">
        <property name="text">
         <string>4xC19</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>420</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string>IP-PDU阈值设置区</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
